website/components/io-home-card/product-logos.ts

```diff
@@ -1,10 +1,34 @@
 export const productLogos = {
-  boundary: require('@hashicorp/mktg-logos/product/boundary/logomark/color.svg?include'),
-  consul: require('@hashicorp/mktg-logos/product/consul/logomark/color.svg?include'),
-  nomad: require('@hashicorp/mktg-logos/product/nomad/logomark/color.svg?include'),
-  packer: require('@hashicorp/mktg-logos/product/packer/logomark/color.svg?include'),
-  terraform: require('@hashicorp/mktg-logos/product/terraform/logomark/color.svg?include'),
-  vagrant: require('@hashicorp/mktg-logos/product/vagrant/logomark/color.svg?include'),
-  vault: require('@hashicorp/mktg-logos/product/vault/logomark/color.svg?include'),
-  waypoint: require('@hashicorp/mktg-logos/product/waypoint/logomark/color.svg?include'),
+  boundary: {
+    color: require('@hashicorp/mktg-logos/product/boundary/logomark/color.svg?include'),
+    neutral: require('@hashicorp/mktg-logos/product/boundary/logomark/white.svg?include'),
+  },
+  consul: {
+    color: require('@hashicorp/mktg-logos/product/consul/logomark/color.svg?include'),
+    neutral: require('@hashicorp/mktg-logos/product/consul/logomark/white.svg?include'),
+  },
+  nomad: {
+    color: require('@hashicorp/mktg-logos/product/nomad/logomark/color.svg?include'),
+    neutral: require('@hashicorp/mktg-logos/product/nomad/logomark/white.svg?include'),
+  },
+  packer: {
+    color: require('@hashicorp/mktg-logos/product/packer/logomark/color.svg?include'),
+    neutral: require('@hashicorp/mktg-logos/product/packer/logomark/white.svg?include'),
+  },
+  terraform: {
+    color: require('@hashicorp/mktg-logos/product/terraform/logomark/color.svg?include'),
+    neutral: require('@hashicorp/mktg-logos/product/terraform/logomark/white.svg?include'),
+  },
+  vagrant: {
+    color: require('@hashicorp/mktg-logos/product/vagrant/logomark/color.svg?include'),
+    neutral: require('@hashicorp/mktg-logos/product/vagrant/logomark/white.svg?include'),
+  },
+  vault: {
+    color: require('@hashicorp/mktg-logos/product/vault/logomark/color.svg?include'),
+    neutral: require('@hashicorp/mktg-logos/product/vault/logomark/white.svg?include'),
+  },
+  waypoint: {
+    color: require('@hashicorp/mktg-logos/product/waypoint/logomark/color.svg?include'),
+    neutral: require('@hashicorp/mktg-logos/product/waypoint/logomark/white.svg?include'),
+  },
 }
```

Comment:
Not a big downside if it's the case, but I believe this causes every logo to be loaded in-browser on every page this is imported?

Just wanted to tap on that; don't change it if that's the case, cause the files will be so small.

Reply:
Acknowledged, idea is this gets replaced by Flight icons once products are included https://github.com/hashicorp/flight/issues/330